doc/user-guides/acrn-shell.rst

```diff
@@ -21,9 +21,12 @@ The ACRN hypervisor shell supports the following commands:
      - List all vCPUs in all VMs
    * - vcpu_dumpreg <vm_id> <vcpu_id>
      - Dump registers for a specific vCPU
-   * - dumpmem <hva> <length>
-     - Dump host memory, starting at a given address, and for a given length
-       (in bytes)
+   * - dump_host_mem <hva> <length>
+     - Dump host memory, starting at a given address(hexadecimal), and for a given length
```

Comment:
How about something like: Dump the host memory region as specified by the start of the region ``hva`` (in hexadecimal) and its length ``length`` (in bytes, decimal number).

Reply:
ok will update